test/plugin/test_out_elasticsearch.rb

```diff
@@ -754,8 +790,9 @@ def test_uses_custom_time_key_format_logs_an_error(tag_for_error)
     ts = "2001/02/03 13:14:01,673+02:00"
     index = "logstash-#{Date.today.strftime("%Y.%m.%d")}"
 
-    driver.emit(sample_record.merge!('@timestamp' => ts))
-    driver.run
+    driver.run(default_tag: 'test') do
+      driver.feed(sample_record.merge!('@timestamp' => ts))
+    end
 
     log = driver.instance.router.emit_error_handler.log
```

Comment:
This test case should confirm whether or not occurring error events.
In v0.12, we can use `driver.instance.router.emit_error_handler.log` to confirm error events.
But, in v0.14, `driver.instance.router.emit_error_handler.log` is removed.
We should another way to confirm error occurred.

In the above my code snippets, I tried to confirm `emit_error_event` method invocation with mock.